recipe/multitenancy/multitenancyClaims.go

```diff
@@ -0,0 +1,84 @@
+package multitenancy
+
+import (
+	"errors"
+	"time"
+
+	"github.com/supertokens/supertokens-golang/recipe/multitenancy/multitenancyclaims"
+	"github.com/supertokens/supertokens-golang/recipe/session/claims"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func NewMultitenancyClaims() (*claims.TypeSessionClaim, *claims.TypeSessionClaim, multitenancyclaims.TypeMultitenancyClaimValidators) {
+	fetchTenantId := func(userId string, userContext supertokens.UserContext) (interface{}, error) {
+		instance, err := GetRecipeInstanceOrThrowError()
+		if err != nil {
+			return nil, err
+		}
+		tenantIdRes, err := instance.GetTenantIdForUserID(userId, userContext)
+		if err != nil {
+			return false, err
+		}
+		if tenantIdRes.OK != nil {
+			return tenantIdRes.OK.TenantId, nil
+		} else {
+			return "", errors.New("UNKNOWN_USER_ID")
+		}
+	}
+
+	fetchDomains := func(userId string, userContext supertokens.UserContext) (interface{}, error) {
+		instance, err := GetRecipeInstanceOrThrowError()
+		if err != nil {
+			return nil, err
+		}
+		tenantIdRes, err := instance.GetTenantIdForUserID(userId, userContext)
+		if err != nil {
+			return false, err
+		}
+		if tenantIdRes.OK != nil {
+			tenantId := tenantIdRes.OK.TenantId
+			domains, err := instance.GetDomainsForTenantId(tenantId, userContext)
+			if err != nil {
+				return nil, err
+			}
+			return domains, nil
+		} else {
+			return nil, errors.New("UNKNOWN_USER_ID")
+		}
+	}
+
+	var defaultMaxAge int64 = 300
+	mtDomainClaim, arrayClaimValidators := claims.PrimitiveArrayClaim("st-tenant-domains", fetchDomains, &defaultMaxAge)
+	mtTenantIdClaim, _ := claims.PrimitiveClaim("st-tenantid", fetchTenantId, &defaultMaxAge)
```

Comment:
we want to have just one claim called `st-tenant`

Reply:
Have called it `st-tenant-domains`, other claim is removed.